cloud/jenkins/pxc_operator_aws_openshift-4.groovy

```diff
@@ -4,6 +4,85 @@ void IsRunTestsInClusterWide() {
     }
 }
 
+void CreateCluster( String CLUSTER_SUFFIX ){
+    
+    withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'openshift-cicd'], file(credentialsId: 'aws-openshift-41-key-pub', variable: 'AWS_NODES_KEY_PUB'), file(credentialsId: 'openshift4-secrets', variable: 'OPENSHIFT_CONF_FILE')]) {
+        sh """
+            platform_version=`echo "\${params.PLATFORM_VER}" | awk -F. '{ printf("%d%03d%03d%03d\\n", \$1,\$2,\$3,\$4); }';`
+            version=`echo "4.12.0" | awk -F. '{ printf("%d%03d%03d%03d\\n", \$1,\$2,\$3,\$4); }';`
```

Comment:
Why is it fixed to `4.12.0`?

Reply:
Because we should update these params only on version 4.12 and more. Older version should use old vars.

Reply:
Thanks!